### PR TITLE
feat: add translation of README.md into French

### PR DESCRIPTION
**Pull request by webdevbynight**
## Summary

Add translation into French of `README.md`and update links to include the French translation.
- **New translation:** add `README.fr.md` file with translation into French of the `README.md` file.
- **Update links:** update `README.md`, `README.hi.md`, `README.ja.md`, `README.kr.md`, `README.pt-BR.md`, `README.ru.md`, `README.uk.md`, `README.zh-CN.md` and `README.zh-TW.md` files to include a link to the French translation.

### CODE REVIEW

**Comment by ematipico** [comment]
I think @Conaclos is the best person for this review :D

File: packages/@biomejs/biome/README.fr.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+</div>
+
+<br>
+
+**Biome** est un ensemble d’outils performant pour des projets web, visant à fournir au développeur des outils pour maintenir la santé desdits projets.
+
+**Biome est un [outil de formatage rapide](./benchmark#formatting)** pour _JavaScript_, _TypeScript_, _JSX_, _JSON_, _CSS_ et _GraphQL_ qui atteint **[97 % de compatibilité avec _Prettier_](https://console.algora.io/challenges/prettier)**.
+
+**Biome est un [outil de linting performant](https://github.com/biomejs/biome/tree/main/benchmark#linting)** pour _JavaScript_, _TypeScript_, _JSX_, _CSS_ et _GraphQL_ qui comprend **[plus de 270 règles](https://biomejs.dev/fr/linter/rules/)** d’ESLint, de typescript-eslint, et [d’autres sources](https://github.com/biomejs/biome/discussions/3).
+Il **sort des diagnostics détaillés et contextualisés** qui vous aident à améliorer votre code et à devenir un meilleur programmeur !
```
Comment (Conaclos) [inline packages/@biomejs/biome/README.fr.md:45]:
```suggestion
Il **forunit des diagnostics détaillés et contextualisés** qui vous aident à améliorer votre code et à devenir un meilleur programmeur !
```

File: packages/@biomejs/biome/README.fr.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+**Biome** est conçu dès le départ pour être utilisé [dans un éditeur de manière interactive](https://biomejs.dev/fr/guides/integrate-in-editor/).
+Il peut formater et linter du code malformé pendant que vous l’écrivez.
```
Comment (Conaclos) [inline packages/@biomejs/biome/README.fr.md:48]:
```suggestion
Il peut formater et analyser du code malformé pendant que vous l’écrivez.
```

File: packages/@biomejs/biome/README.fr.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+## Documentation
+
+Consultez notre [page d’accueil][biomejs] pour en savoir plus sur Biome
+ou allez directement sur le [guide de démarrage][getting-started] pour commencer à utiliser Biome.
+
+## Plus sur Biome
+
+**Biome** a des réglages par défaut robustes et ne demande pas de configuration.
+
+**Biome** vise à prendre en charge [tous les principaux langages][language-support] du développement web moderne.
```
Comment (Conaclos) [inline packages/@biomejs/biome/README.fr.md:83]:
```suggestion
**Biome** vise à prendre en charge [les principaux langages][language-support] du développement web moderne.
```

File: packages/@biomejs/biome/README.fr.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+**Biome** [ne demande pas Node.js](https://biomejs.dev/fr/guides/manual-installation/) pour fonctionnner.
```
Comment (Conaclos) [inline packages/@biomejs/biome/README.fr.md:85]:
```suggestion
**Biome** [ne requiére pas Node.js](https://biomejs.dev/fr/guides/manual-installation/) pour fonctionnner.
```

File: packages/@biomejs/biome/README.fr.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+**Biome** a une prise en charge de LSP de premier choix, avec un analyseur sophistiqué qui représente le texte source en toute fidélité et une excellente récupération des erreurs.
```
Comment (Conaclos) [inline packages/@biomejs/biome/README.fr.md:87]:
```suggestion
**Biome** a une prise en charge du LSP de premier ordre, avec un analyseur sophistiqué qui représente le code source en toute fidélité et une excellente récupération des erreurs.
```

File: packages/@biomejs/biome/README.fr.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+
+### Sponsoring et financement du projet
+
+Vous pouvez sponsoriser ou financer le projet via [Open collective](https://opencollective.com/biome) ou [GitHub sponsors](https://github.com/sponsors/biomejs).
+
+Biome offre un programme de sponsoring simple qui permet aux entreprises d’obtenir de la visibilité et la reconnaissance de divers développeurs.
+
+### Financement d’un problème
+
+Nous utilisons [Polar.sh](https://polar.sh/biomejs) pour voter pour des fonctionnalités spécifiques que vous aimeriez voir implémentées et pour les promouvoir. Jetez un œil à notre carnet de commandes et aidez-nous.
```
Comment (Conaclos) [inline packages/@biomejs/biome/README.fr.md:107]:
```suggestion
Nous utilisons [Polar.sh](https://polar.sh/biomejs) pour voter pour des fonctionnalités spécifiques que vous aimeriez voir implémentées et pour les promouvoir. Jetez un coup d'œil aux fonctionnalités demandées et aidez-nous à les financer.
```

**Comment by Conaclos** [comment]
Merci pour ta contribution ! J'ai laissé quelques commentaires / suggestions :)

File: packages/@biomejs/biome/README.fr.md (hunk 1 above)
Comment (webdevbynight) [reply packages/@biomejs/biome/README.fr.md:45]:
```suggestion
Il **fournit des diagnostics détaillés et contextualisés** qui vous aident à améliorer votre code et à devenir un meilleur programmeur !
```

File: packages/@biomejs/biome/README.fr.md (hunk 4 above)
Comment (webdevbynight) [reply packages/@biomejs/biome/README.fr.md:85]:
```suggestion
**Biome** [ne requiert pas Node.js](https://biomejs.dev/fr/guides/manual-installation/) pour fonctionnner.
```

**Pull request merged**